docs/packages/xstate-react/index.md

```diff
@@ -208,6 +208,29 @@ const App = ({ service }) => {
 };
 ```
 
+::: warn
+The default comparison is a strict reference comparison (`===`). If your selector returns non-primitive values, such as objects or arrays, you should keep this in mind and either return the same reference, or provide a shallow or deep comparator.
```

Comment:
I was wondering what React Redux does - since `connect` was always using a `shallowEqual` under the hood - but it seems that they have "switched" to using `===` when they have introduced `useSelector`:
https://github.com/reduxjs/react-redux/blob/720f0ba79236cdc3e1115f4ef9a7760a21784b48/src/hooks/useSelector.ts#L35

Reply:
I'm now doing what React-Redux does 👍